scss/components/_activity-feed.scss

```diff
@@ -139,10 +139,13 @@
 }
 
 .civicase__activity-feed__list-container__left {
+  min-width: 620px;
+  overflow: auto;
   width: 50%;
 }
 
 .civicase__activity-feed__list-container__right {
+  min-width: 425px;
```

Comment:
where does the value `425px` and `620px` come from?

Reply:
in any case they seem pretty important as they define the main layout, shouldn't they be in variables

Reply:
Jon is defining these `min` and `max` heights in C51-326. Can we keep this as it is as I dont have time left today and also as it will be refactored very soon.